.github/workflows/apolo-build.yaml

```diff
@@ -0,0 +1,70 @@
+name: apolo-build
+on:
+  workflow_dispatch:
+  push:
+    branches: [master]
+    tags:
+      - 'v*'
+  pull_request:
+    branches: [master]
+env:
+  REGISTRY: ghcr.io
+  FULL_IMAGE_NAME: ghcr.io/neuro-inc/speaches
```

Comment:
if upstreap accepts our suggestions, could we go with their image?

Reply:
Great, let's keep focus on this case and send them PR. So we will not need to keep our own copy of the repo (at least, for now)